Description: Bump dependency versions to match debian unstable
 This crate has several dependencies on older versions making it uninstallable
 in unstable. No issue has been seen bumping these to current versions.
Author: Loren M. Lang <lorenl@north-winds.org>, Matthias Geiger <werdahias@debian.org>, Peter Michael Green <plugwash@debian.org>
Forwarded: not-needed
Last-Update: 2024-10-05
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/

Index: adblock/Cargo.toml
===================================================================
--- adblock.orig/Cargo.toml
+++ adblock/Cargo.toml
@@ -45,16 +45,16 @@ optional = true
 default-features = false
 
 [dependencies.base64]
-version = "0.13"
+version = "0.22"
 
 [dependencies.bitflags]
 version = "1.3"
 
 [dependencies.idna]
-version = "0.2"
+version = ">= 0.2"
 
 [dependencies.itertools]
-version = "0.10"
+version = ">= 0.10, < 1.0"
 
 [dependencies.lifeguard]
 version = "^ 0.6.1"
@@ -73,10 +73,10 @@ version = "2.1"
 version = "1.5"
 
 [dependencies.rmp-serde]
-version = "0.15"
+version = "1"
 
 [dependencies.seahash]
-version = "3"
+version = "4"
 
 [dependencies.serde]
 version = "1.0"
@@ -105,7 +105,7 @@ version = "1"
 version = "0.3"
 
 [dev-dependencies.mock_instant]
-version = "0.5"
+version = "0.3"
 
 [dev-dependencies.reqwest]
 version = "0.11"
@@ -116,7 +116,7 @@ default-features = false
 version = "1.0"
 
 [dev-dependencies.sha2]
-version = "0.9"
+version = "0.10"
 
 [dev-dependencies.tokio]
 version = "1.24"
